autokeras/__init__.py

```diff
@@ -4,5 +4,5 @@
 
 from autokeras.net_module import CnnModule, MlpModule
 
-from autokeras.pretrained.object_detector import ObjectDetector
-from autokeras.pretrained.face_detector import FaceDetector
+# from autokeras.pretrained.object_detector import ObjectDetector
```

Comment:
We need these two lines.
The user should be able to directly import the pretrained model from autokeras.something.

Reply:
I moveed this to pretrained __init__.py . My feeling is that the pretrained should be independent to other functions. When load current pretrain model, one can use autokeras.pretrained.something.